react-native-tileview/src/main/java/com/example/react_native_tileview/TileviewManager.java

```diff
@@ -0,0 +1,55 @@
+package com.example.react_native_tileview;
+
+import android.app.Activity;
+
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.SimpleViewManager;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.uimanager.ThemedReactContext;
+import com.facebook.react.uimanager.annotations.ReactProp;
+
+
+import com.qozix.tileview.TileView;
+
+
+
+public class TileviewManager extends ReactContextBaseJavaModule {
+
+    public static final String REACT_CLASS = "Tileview";
+    private Activity mActivity = null;
+    private ThemedReactContext mContext = null;
+
+    @Override
+    public String getName() {
+        return REACT_CLASS;
+    }
+
+    public TileviewManager(ReactApplicationContext reactContext) {
+        super(reactContext);
+    }
+
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    @ReactProp(name = "displayTiles")
+    protected void displayTiles(TileView tileView, String filePath) {
+
+        // size of original image at 100% mScale
+        tileView.setSize(4015, 4057);
+
+        // we're running from assets, should be fairly fast decodes, go ahead and render asap
+        tileView.setShouldRenderWhilePanning(true);
+
+        // detail levels
+        tileView.addDetailLevel(1.000f, filePath.concat("/6/%d/%d.jpg"));
```

Comment:
Where does `filePath` get the root directory of the `Documents` folder prepended to it? See the [ios map view](https://github.com/WWU-iCharts/iCharts/blob/master/ios/IChartsMap.m#L37) where the application document directory is used to build the url: `{documentsDir}/{regionId}/{z}/{x}/{y}.png`